External/FEXCore/Source/Interface/Core/Frontend.h

```diff
@@ -58,6 +58,7 @@ class Decoder final {
   bool DecodeInstruction(uint64_t PC);
 
   void BranchTargetInMultiblockRange();
+  bool BranchTargetCanContinue(bool FinalInstruction);
```

Comment:
```suggestion
  bool BranchTargetCanContinue(bool FinalInstruction) const;
```

This seems like it can be const, though disregard if it turns out to not be the case.

Reply:
You're right, resolved.